data/src/main/java/com/example/data/Data.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.data
+
+import android.util.Log
+import com.example.domain.Base
+import com.example.domain.Item
+import com.google.gson.GsonBuilder
+import retrofit2.*
+import retrofit2.converter.gson.GsonConverterFactory
+
+class Data {
+    var repo: MutableMap<String, List<Base>> = mutableMapOf()
+
+    private fun loadData(item: String) {
+        var retrofit: Retrofit = Retrofit.Builder()
+            .baseUrl("https://raw.githubusercontent.com/karl-park/com.ninetynine.healthysalad/master/server/")
+            .addConverterFactory(GsonConverterFactory.create(GsonBuilder().create()))
+            .build()
+        var service: DataGetter = retrofit.create(DataGetter::class.java)
+        var call: Call<Item> = service.listItems(item)
+        call.enqueue(object : Callback<Item> {
+            override fun onFailure(call: Call<Item>, t: Throwable) {
+                Log.d("Called", t.toString())
+            }
+
+            override fun onResponse(call: Call<Item>, response: Response<Item>) {
+                response.body()?.body?.data?.base?.forEach { item -> Log.d("Called", item.name) }
+                onLoadSuccess(response, item)
+            }
+        })
+    }
+
+    private fun loadItem(item: String) {
+        val fileName = "$item.json"
+        loadData(fileName)
+    }
+
+    fun loadAllItems() {
+        var ingredients = listOf<String>("base", "crunchy", "dressing", "protein", "soft")
+        for (item in ingredients) {
+            loadItem(item)
+        }
+    }
+
+    fun onLoadSuccess(response: Response<Item>, item: String) {
+        var data: List<Base> = response.body()?.body?.data?.base ?: emptyList()
```

Comment:
I think "data" is an immutable variable.
why you declare this variable as a mutable variable?
if you don't have any specific reason, please use "val" rather than "var".